cmd/describe.go

```diff
@@ -101,6 +108,17 @@ var describeHeadCmd = &cobra.Command{
 		}
 
 		m = m.ApplyFilters(&lib.FilterOptions{Name: name, Fuzzy: fuzzy})
+
+		mods := m.Modules
+
+		if dependents {
```

Comment:
It would be good if we could move this check inside `ApplyFilters`. That way the logic is encapsulated in one place and `ApplyFilters` tests can verify implementation.

Reply:
I think changing the signature of `ApplyFilter` would be more appropriate.